### ISSUE TITLE
can I set annother value of batchsize besides 1?

### ISSUE THREAD

**Issue by iwanli** [chenyuntc/simple-faster-rcnn-pytorch#91]
I want to set a bigger value of batch size,but it can not work when i change the value,it seems the value can only be 1.How can i set a bigger value? thank you!

**Comment by howardyclo** [comment]
@chenyuntc Hello, I would also love to know this.

**Comment by ZhuYingJessica** [comment]
@iwanli @howardyclo   Hello, have you solved this problem? I'll be grateful if you can give me some suggestions

**Comment by chrisway613** [comment]
it will take some works if u wanna train with batch size > 1.
First, u should change the code for data processing, cuz each image has different number of objects, the ground truth label cannot be form a batch data directly, u can fill some 'non-objects' label in a batch, which is like the YOLO do;
Second, u should change the code in the training part, mainly in the forward() function in class 'FasterRCNNTrainer';
Then, u can test your code and get more details.
